<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_Acc">
      <property name="text">
       <string>Waiting for WT901...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_Angle">
      <property name="text">
       <string>Angle Data</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_SimStatus">
      <property name="text">
       <string>GPS Status</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textConsole"/>
    </item>
    <item>
     <widget class="QPushButton" name="btnStart">
      <property name="text">
       <string>START</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnStop">
      <property name="text">
       <string>STOP</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
